resources/views/platform/manage-hashes-page.blade.php

```diff
@@ -0,0 +1,91 @@
+@props([
+    'game',
+    'me',
+])
+
+<?php
+use App\Community\Enums\ArticleType;
+
+$articleTypeGameHash = ArticleType::GameHash;
+$numLogs = getRecentArticleComments($articleTypeGameHash, $game->ID, $logs);
+?>
+
+<x-app-layout pageTitle="{{ 'Manage Game Hashes - ' . $game->Title }}">
+    <div>
+        <x-game.breadcrumbs
+            :targetConsoleId="$game->system->ID"
+            :targetConsoleName="$game->system->Name"
+            :targetGameId="$game->ID"
+            :targetGameName="$game->Title"
+            currentPageLabel="Manage Hashes"
+        />
+
+        <div class="mt-3 mb-1 w-full flex gap-x-3">
+            {!! gameAvatar($game->toArray(), label: false, iconSize: 48, iconClass: 'rounded-sm') !!}
+            <h1 class="mt-[10px] w-full">Manage Hashes</h1>
+        </div>
+
+        <div class="mb-4">
+            <x-game.link-buttons.index
+                :allowedLinks="['forum-topic', 'game-files']"
+                :gameId="$game->ID"
+                :gameTitle="$game->Title"
+                :gameForumTopicId="$game->ForumTopicID"
+                variant="row"
+            />
+        </div>
+
+        <div class="mb-6 flex gap-x-4">
+            <x-alert variant="destructive">
+                <x-slot name="title">Warning</x-slot>
+                <x-slot name="description">
+                    <div class="flex flex-col">
+                        <p>
+                            PLEASE be careful when using this tool. Mistakes can cause a lot of tickets to be created.
+                        </p>
+
+                        <p>
+                            If you're not <span class="underline font-semibold">100% sure</span> of what you're doing,
+                            <a href="{{ route('message.create') . '?to=RAdmin&subject=Attempt+to+Unlink+' . urlencode($game->Title) . '&message=%5Bgame=' . $game->ID . '%5D' }}">
```

Comment:
I wonder if the subject line for this message should be changed. This page allows for so much more than just unlinking.

Reply:
I agree, that would make more sense. Changed in latest to:

"Help with Hash Management for {{ gameTitle }}"